tests/std/tests/Dev11_0535636_functional_overhaul/test.cpp

```diff
@@ -2169,6 +2159,42 @@ void test_more_bind() {
     STATIC_ASSERT(
         is_same_v<decltype(p11), Pack<const Typewriter&, const bool&,
                                      Pack<const Typewriter&, Z&, const Z&, Z&&, const Z&&>&&, Z&, const double&>>);
+
+#if _HAS_CXX20

```

Comment:
Why is this section C++20-only? The "nothrowness" of the library isn't sensitive to language mode AFAICS, so we should test it in all language modes.

Reply:
It can't be tested in all language modes because `is_nothrow_invocable` is available since C++17. It is required since C++20 so I made it C++20-only. I will change it to `_HAS_CXX17`.